docs/concepts/privacy-and-security.md

```diff
@@ -126,7 +126,7 @@ The following example shows how a task pane add-in specifies the  **ReadDocument
 ```xml
 <?xml version="1.0" encoding="utf-8"?>
 <OfficeApp xmlns="http://schemas.microsoft.com/office/appforoffice/1.0"
-           xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance" 
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
```

Comment:
I'm curious why this change was made. I get redirected to the secure URL in a browser.

Reply:
I'm not an XML expert, but I know that XML namespace names aren't necessarily actual URLs. My guess would be that sometime since 2001, the www.w3.org started redirecting http requests to it's domain to https.